packages/sources/vue/src/components/VtmnRadioButton/VtmnRadioButton.vue

```diff
@@ -1,48 +1,50 @@
 <script lang="ts">
 import '@vtmn/css-radio-button/dist/index-with-vars.css';
-import { defineComponent } from 'vue';
+import { defineComponent, PropType } from 'vue';
 
 export default /*#__PURE__*/ defineComponent({
   name: 'VtmnRadioButton',
   props: {
+    modelValue: {
+      type: [String, Number] as PropType<string | number>,
+      default: ''
+    },
```

Comment:
Are you sure that's it's the best practice to call this "modelValue"?

Reply:
Yes, it's a reserved props name

See more details here : https://v3.vuejs.org/guide/migration/v-model.html

Reply:
@ThomasLaigneauDecathlon as we discussed, so you will remove this?